src/redash_stmo/data_sources/health.py

```diff
@@ -88,16 +95,25 @@ def test_connection(self, custom_query_text=None):
 @require_super_admin
 def stmo_status_api():
     status = original_get_status()
-    status['data_sources'] = json.loads(redis_connection.get('data_sources:health') or '{}')
+    health_data = json.loads(redis_connection.get('data_sources:health') or '{}')
+
+    # Get the top level status for each data source
+    for data_source_id in health_data:
```

Comment:
Would you mind iterating over the values here so we don't have to do dict lookups?
```suggestion
    for health_data_point in health_data.values():
        dashboard_label = "[Data Source Health] {name}".format(name=health_data_point["name"])
        status[dashboard_label] = health_data_point["status"]
```